### PR TITLE
Fix raw & some fixes🌵 

### PR DESCRIPTION
**Pull request by Mrgaton**

### CODE REVIEW

File: src/routes/v1/access.route.ts [hunk 1]
```diff
@@ -46,7 +46,7 @@ export default new Elysia({
 		{
 			params: t.Object(
 				{
-					key: t.String({
+					key: t.Any({
```
Comment (tnfAngel) [inline src/routes/v1/access.route.ts:49]: Why

**Review by tnfAngel — CHANGES_REQUESTED** [review]

File: src/routes/v1/access.route.ts (hunk 1 above)
Comment (Mrgaton) [inline src/routes/v1/access.route.ts]: Perdon claro

Comment (Mrgaton) [reply src/routes/v1/access.route.ts]:
```suggestion
					key: t.String({
```

**Review by tnfAngel — APPROVED** [review]
LGTM for the moment

**Pull request merged**